src/views/network/CreateL2NetworkForm.vue

```diff
@@ -109,6 +109,32 @@
           <a-form-item :label="$t('label.bypassvlanoverlapcheck')" v-if="!this.isObjectEmpty(this.selectedNetworkOffering) && this.selectedNetworkOffering.specifyvlan">
             <a-switch v-decorator="['bypassvlanoverlapcheck']" />
           </a-form-item>
+          <a-form-item :label="$t('label.isolatedpvlantype')" v-if="!this.isObjectEmpty(this.selectedNetworkOffering) && this.selectedNetworkOffering.specifyvlan">
+            <a-radio-group
+              v-decorator="['isolatedpvlantype', {
+                initialValue: this.isolatePvlanType
+              }]"
+              buttonStyle="solid"
+              @change="selected => { this.handleIsolatedPvlanTypeChange(selected.target.value) }">
```

Comment:
minor: can be simplified
```suggestion
              @change="selected => { this.isolatePvlanType = selected.target.value }">
```